build/template-build-and-run-all-tests.yaml

```diff
@@ -107,19 +109,19 @@ jobs: #Build and stage projects
       DataFileDirectory: ${{ parameters.DataFileDirectory }}
 
 # APP CENTER IOS MSAL TESTS PHASE
```

Comment:
can these be removed if not needed?

Reply:
This is temporary to get the package out. Still need to make final decision about testing on the App Center